### ISSUE TITLE
Lower Multinomial without Replacement

### ISSUE THREAD

**Issue by cowanmeg** [pytorch/xla#4865]
## 🚀 Feature
Lower multinomial without replacement.

## Motivation

Currently, we lower multinomial with replacement (https://github.com/pytorch/xla/issues/4839), but we want to accelerate both paths.

Some relevant discussion on tensorflow that might be helpful: https://github.com/tensorflow/tensorflow/issues/9260

**Comment by JackCaoG** [comment]
Can this be closed?

**Comment by cowanmeg** [comment]
No, the current implementation falls back to the cpu when replacement is False.

**Comment by wonjoolee95** [comment]
@cowanmeg, I'll assign this to you as part of cleaning up our unassigned issues. No immediate need to work on this as far as I can tell, so feel free to put it in your backlog and prioritize as necessary.
